### PR TITLE
Install OpenBLAS manually

### PR DESCRIPTION
**Pull request by Rombur**

### CODE REVIEW

**Comment by Rombur** [comment]
This seems to be working. The trick was to use `DYNAMIC_ARCH=1`.

**Comment by aprokop** [comment]
I'm going to test the container with Lanczos branch now.

File: ci/Dockerfile [hunk 1]
```diff
@@ -235,6 +262,10 @@ RUN export DEAL_II_HASH=9cd42e465767c5a8ceffddbbc17f6737503337ac && \
     rm -rf ${DEAL_II_SOURCE_DIR}
 ENV DEAL_II_DIR=${INSTALL_DIR}/dealii
 
+# deal.II does not export the include directory for BLAS/LAPACK, so add it to
+# CPATH
+ENV CPATH=${OPENBLAS_DIR}/include:${CPATH}
```
Comment (dalg24) [inline ci/Dockerfile]: :/

Comment (aprokop) [reply ci/Dockerfile]: Yeah, that. May be a bug in DealII.

**Pull request merged**